templates/job/detail.html

```diff
@@ -185,6 +185,16 @@ <h1 class="min-w-0 text-3xl break-words md:text-4xl font-bold text-slate-900">
           -
           {% endif %}
         {% /description_item %}
+
+        {% #description_item title="Git commit hash" stacked=True %}
+          {% link href=job.job_request.get_repo_url text=job.job_request.sha|slice:7|default:"-" %}
+        {% /description_item %}
+
+        {% if compare_url %}
+          {% #description_item title="Compare code" stacked=True %}
```

Comment:
Does this need to say something more explicit like "show code differences between this run and the last one"? (but put more elegantly!)

Reply:
yes, certainly could be made clearer, I'll have a think of some suitable wording

Reply:
I've made a couple of suggestions in a separate comment. I think that on this page it's especially important to emphasize that _all_ the code is being compared, not just the code that's relevant to a job. (I'm not really sure whether this page should have this link, as we know the relationship between a job and a job request can confuse users.)